setup.py

```diff
@@ -92,6 +92,8 @@ def get_extras_require() -> Dict[str, List[str]]:
         ],
         "integration": [
             "allennlp>=2.2.0 ; python_version>'3.6'",
+            # TODO(c-bata): Remove cached-path after allennllp supports v1.1.3
+            "cached-path>=1.1.2 ; python_version>'3.6'",
```

Comment:
The pull request to resolve this issue is already created at https://github.com/allenai/allennlp/pull/5665.